app-config.yaml

```diff
@@ -215,12 +215,18 @@ catalog:
         serviceAccountToken: ${moc_infra_token}
         owner: janus-authors
 
-    # Plugin: GitHub Discovery & Org Data
+    # Plugin: GitHub Org Data
     githubOrg:
       default:
         id: production
         orgUrl: ${GITHUB_ORG_URL}
```

Comment:
Can this be a list of organizations?

Reply:
`orgUrl` can't be a list.

theoretically based on https://github.com/janus-idp/backstage-showcase/blob/83003f013c0a3c0cf0e8de9b6bed2eccdd5594b2/packages/backend/src/plugins/catalog.ts#L75-L94
it should be possible to specify multiple orgs like this:
```yaml
    githubOrg:
      org1:
        id: production-org1
        orgUrl: org1
      org2:
        id: production-org2
        orgUrl: org2
```

I haven't tested it. It looks weird to me, and I don't know if it can deal with conflicting items.

There is `GithubMultiOrgEntityProvider`, which we could use instead of `GithubOrgEntityProvider`. Based on docs, this is how to ingest data from multiple orgs. This one will namespace groups according to the org they originate from to avoid potential name duplicates.


If we want to change to `GithubMultiOrgEntityProvider` I can look into it in a separate PR. I would like to keep this PR documentation only.

Reply:
I've tested both  `GithubOrgEntityProvider` and `GithubMultiOrgEntityProvider`.
Currently, we use `GithubOrgEntityProvider` and it is possible to specify multiple groups as I described in previous comment.

Let's say that we have two orgs that have the same group called `owners`.
`GithubOrgEntityProvider` will import only one group called `owners`, and then it will complain about duplicated groups when trying to import the second one.

I've also tested this with `GithubMultiOrgEntityProvider` and this will import both groups as `org1/owners` and `org2/owners`.